ui/src/app/views/workspace-edit.tsx

```diff
@@ -627,8 +629,9 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               sufficiently detailed responses at a 5th grade reading level.  Your responses
               will not be used to make decisions about data access. <br/> <br/>
               <i>Note that you are required to create separate Workspaces for each project
-                for which you access AoU data, hence the responses below are expected to be specific
-                to the project for which you are creating this particular Workspace.</i> </div>
+                for which you access All of Us data, hence the responses below are expected
```

Comment:
Does this 'All of Us' also have to be italics

Reply:
i could unitalicize it since the text itself is already in italics

Reply:
Ahh ok thats fine i did not notice the <i> tag